DataFormats/TrackerRecHit2D/interface/VectorHit.h

```diff
@@ -0,0 +1,163 @@
+#ifndef TrackerRecHit2D_VectorHit_h
+#define TrackerRecHit2D_VectorHit_h
+
+/** \class VectorHit
+ *
+ * 4-parameter RecHits for Phase2 Tracker (x,y, dx/dz, dy/dz)
+ *
+ * $Date: 2015/03/30 $
+ * \author Erica Brondolin
+ *
+ */
+
+#include "DataFormats/TrackerRecHit2D/interface/BaseTrackerRecHit.h"
+#include "DataFormats/TrackerRecHit2D/interface/VectorHit2D.h"
+#include "DataFormats/TrackerRecHit2D/interface/OmniClusterRef.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/Common/interface/DetSetVectorNew.h"
+
+#include "DataFormats/Phase2TrackerCluster/interface/Phase2TrackerCluster1D.h"
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+#include "Geometry/CommonDetUnit/interface/PixelGeomDetUnit.h"
+#include "MagneticField/Engine/interface/MagneticField.h"
```

Comment:
hopefully an unneeded #include.

Reply:
Actually this include is needed as the magnetic field is used in two member functions of VectorHit to get momentum and transverse momentum estimate from the estimated curvature from the VH. Is there a better way to access it that does not require this include?